pkg/alertmanager/alertspb/alerts.proto

```diff
@@ -32,3 +32,15 @@ message FullStateDesc {
 
   clusterpb.FullState state = 1;
 }
+
+message GrafanaAlertConfigDesc {
+  // Alertmanager (clusterpb) types do not have Equal methods.
+  option (gogoproto.equal) = false;
+
+  string user = 1;
+  string raw_config = 2;
+  int64 id = 3;
+  string hash = 4;
+  int64 created_at = 5;
```

Comment:
nit: again a suggestion to add the unit to the field name to make it more obivous what it stores. You can also use a Timestamp type like [the ruler does](https://github.com/grafana/mimir/blob/aff210a3b1c2399d3f7c01abec7aab36378d0a11/pkg/ruler/ruler.proto#L59) if you want to be super clear.

Reply:
Solved in [083abea](https://github.com/grafana/mimir/pull/6682/commits/083abea0147e15612272e7bfab850550f649e3db)